modules/custom-post-types/portfolios.php

```diff
@@ -713,6 +723,19 @@ static function get_project_tags( $post_id ) {
 	}
 
 	/**
+	 * Displays the author of the current portfolio project.
+	 *
+	 * @return html
+	 */
+	static function get_project_author() {
+		$html = '<div class="project-author"><span>' . esc_html__( 'Author:', 'jetpack' ) . '</span> ';
```

Comment:
Good catch @jeherve, thanks! You're right, including the author name in the translatable string is the way to go here. @eliorivero if you see i18n strings split like that somewhere else, feel free to create a new issue with the i18n label :)